_translations/en.yml

```diff
@@ -751,6 +751,16 @@
     offlinemultitxt: "Bitcoin includes a multi-signature feature that allows a transaction to require multiple independent approvals to be spent. This can be used by an organization to give its members access to its treasury while only allowing a withdrawal if 3 of 5 members sign the transaction. Some web wallets also provide multi-signature wallets, allowing the user to keep control over their money while preventing a thief from stealing funds by compromising a single device or server."
     offlinetestament: "Think about your testament"
     offlinetestamenttxt: "Your bitcoins can be lost forever if you don't have a backup plan for your peers and family. If the location of your wallets or your passwords are not known by anyone when you are gone, there is no hope that your funds will ever be recovered. Taking a bit of time on these matters can make a huge difference."
+  spend-bitcoin:
+    title: "Spending Bitcoin - Bitcoin"
+    pagetitle: "Spending Bitcoin"
+    summary: "There are thousands of businesses, across the globe, that accept Bitcoin."
+    products-online: "Find products for sale online"
+    products-online-text: "One common use for Bitcoin is making purchases online. There are hundreds of online shops and retailers that accept Bitcoin. Using a <a href=\"https://spendabit.co/\">search-engine like Spendabit</a> you can search through millions of products, all available for purchase with bitcoins."
+    directory: "Navigate a business directory"
+    directory-text: "You can also find many businesses <a href=\"http://usebitcoins.info/\">listed in online directories</a>. (<strong>Note:</strong> Unfortunately this directory has not been well-maintained and presently includes <em>many broken links</em> and references to defunct businesses.)"
```

Comment:
Indeed. On the one hand it looks bad on Bitcoin.org's part to link to a site that's so poorly maintained; on the other hand we kinda need something there. Maybe if we just tone down the warning? E.g., "(Unfortunately this directory includes a number of links to defunct businesses and old websites, at present.)"